### DISCUSSION TITLE
Is there an API to retrieve InChI atom order for an RDKit molecule?

### DISCUSSION THREAD

**Discussion by coleb** [rdkit/rdkit#4629] [Q&A]
Has anyone tried to reproduce this paper in RDKit yet? 
https://jcheminf.biomedcentral.com/articles/10.1186/1758-2946-4-22

For my particular use case, just getting the InChI atom order would be useful by itself. Thank you!

**Comment by greglandrum** [comment]
There's nothing in the RDKit for this. I'm not sure if anyone else has done it outside the RDKit though.
